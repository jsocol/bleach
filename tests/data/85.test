<div id="14"><input pattern=^((a+.)a)+$ value=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa!>//["'`-->]]>]</div>
