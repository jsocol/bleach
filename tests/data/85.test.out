<div id="14"><input value="aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa!" pattern="^((a+.)a)+$">//["'`--&gt;]]&gt;]</div>
